tests/phpunit/tests/interactivity-api/wpInteractivityAPI.php

```diff
@@ -748,14 +748,17 @@ public function test_process_directives_does_not_change_inner_html_in_math() {
 	 * @param string $directive_value The directive attribute value to evaluate.
 	 * @return mixed The result of the evaluate method.
 	 */
-	private function evaluate( $directive_value ) {
-		$generate_state = function ( $name ) {
+	private function evaluate( $directive_value, ) {
```

Comment:
Trailing comma may be a problem in some supported PHP versions.
```suggestion
	private function evaluate( $directive_value ) {
```

Reply:
it is being indeed, thanks! I was getting crazy